CONTRIBUTING.md

```diff
@@ -165,4 +166,16 @@ Please keep in mind that the team may **suspend or reject** pull requests. There
 -   Poor quality code.
 -   Other [development priorities](https://testcafe.io/402949/roadmap) may take precedence.
 
-Please also note that we do not merge pull requests until the changes are documented.
+Please also note that we do not merge pull requests until the changes are documented. You're welcome to document a new capability yourself. You should add your description as a pull request comment, and include links to Testcafe [documentation](https://testcafe.io/documentation) pages that need editing. While writing the documentation, please, follow the [guidelines](#guidelines).
+
+## Contribute to Documentation
+
+Testcafe [documentation](https://testcafe.io/documentation) is not open-source. If you find an inaccuracy in the docs, send us a note in a [Github issue](https://github.com/DevExpress/testcafe/issues/new). Please, don't forget to add `AREA: docs` label so that the appropriate team gets a new issue for review.
+
+#### Guidelines
+
+We follow a set of documentation guidelines and practices. The main idea is to make our docs easy to read. So, when submitting a new issue, please, make sure that:
```

Comment:
Who are the "we" in this sentence? I'd rather specify this point, or make 'documentation' the subject of the sentence. "The main idea is" might be too colloquial and definitely doesn't add anything to the paragraph
```suggestion
The technical writing team at Developer Express adheres to a set of guidelines and practices that make our docs easy to read. When you submit a new issue, please, make sure that:
```